src/user-guide/task-implementation/skip-mode.rst

```diff
@@ -0,0 +1,119 @@
+.. _task-run-modes.skip:
+
+Skip Mode
+=========
+
+.. versionadded:: 8.4.0
+
+Skip mode is designed as an aid to workflow control:
+
+* It allows creation of :term:`dummy tasks<dummy task>` as part of workflow
+  design.
+* It allows skipping of tasks in a running workflow using either:
+  * ``cylc broadcast -s 'run mode = skip'`` (for when it is ready to run)
+  * ``cylc set --out skip`` (to immediately skip)
+
+.. note::
+
+   Setting :cylc:conf:`[runtime][<namespace>]run mode=skip` in your ``flow.cylc``
+   will lead to ``cylc validate`` returning a warning::
+
+      WARNING - The following tasks are set to run in skip mode:
+        * example_task
+
+   This is designed to prevent users running a task in skip mode by mistake.
+   If you are using skip mode deliberately then this can be
+   ignored.
+
+Task Outputs
+^^^^^^^^^^^^
+
+Skip mode allows the user to specify which task outputs
+will be emitted using :cylc:conf:`[runtime][<namespace>][skip]outputs`.
+
+By default:
+
+* All required outputs will be generated.
+* ``succeeded`` will be generated even if success is :term:`optional <optional output>`.
+* If ``[skip]outputs`` is specified and does not include either ``succeeded``
```

Comment:
This will insert a link to the config, please test.

```suggestion
* If `[skip]outputs <flow.cylc[runtime][<namespace>][skip]outputs>` is specified and does not
   include either ``succeeded``
```